src/main/java/lesson7task2/LettersUppercaseConverter.java

```diff
@@ -0,0 +1,11 @@
+package lesson7task2;
+
+public class LettersUppercaseConverter {
+    public static String ConvertToUppercase(String text) {
+
+        if (text == null) {
+            return "The text is null";
```

Comment:
This is not good approach to do it. Having that if i will pass a null value you will return me a "The text is null". You should return null here or "" (empty string);